src/main/java/seedu/address/model/person/Person.java

```diff
@@ -194,39 +177,37 @@ public int hashCode() {
     public String toString() {
         final StringBuilder builder = new StringBuilder();
 
-        builder.append(getName());
-        if (getPhone() != null) {
-            builder.append(" Phone: ")
-                    .append(getPhone());
-        }
-        if (getEmail() != null) {
-            builder.append(" Email: ")
-                    .append(getEmail());
-        }
-        if (getAddress() != null) {
-            builder.append(" Address: ")
-                    .append(getAddress());
-        }
+        builder.append(getName())
+                .append(personFieldToString(getPhone()))
+                .append(personFieldToString(getEmail()))
+                .append(personFieldToString(getAddress()));
+
         if (getClientSources().size() > 0) {
             builder.append(" ClientSources: ");
             getClientSources().forEach(builder::append);
         }
-        if (getNote() != null) {
-            builder.append(" Note: ")
-                    .append(getNote());
-        }
 
-        if (getPolicy() != null) {
-            builder.append(" Policy: ")
-                    .append(getPolicy());
-        }
-        // Not necessary to add archive status
-
-        builder.append(" Priority: ")
-                .append(getPriority());
+        builder.append(personFieldToString(getNote()))
+                .append(personFieldToString(getPolicy()))
+                .append(personFieldToString(getPriority()));
 
         return builder.toString();
     }
 
+    /**
+     * Returns a formatted string containing the field's class name and the field.
+     * If the field is not present, return an empty string.
+     *
+     * @param field The field in the Person class.
+     * @return String with field's class name and the field.
+     */
+    private String personFieldToString(Object field) {
```

Comment:
Ok im not sure if there is any... like proper name or any code quality guideline behind this. If there is a StringBuilder, we just append everything because it is faster. But if we use " " + string + " ", like adding strings together, this makes the Stringbuilder like abit useless cuz we are going back to the old way of concatenating strings.  (which is alot slower) - wew 2040

Reply:
Great feedback, I didnt like this part too (but I was lazy haha).
Changed in latest commit